docs/SOPs/data_retraction_SOP.md

```diff
@@ -88,6 +109,22 @@ Currently, we store contributor data in the cloud **before** and **while** broke
     **Example**: aws s3 rm -r s3://org-hca-data-archive-upload-prod/<span style="color:red">779ecf45-e930-459e-b87b-c89d3c4546c7/</span>
     
     <span style="color:red">**IMPORTANT NOTE**</span>: Please ensure the path you introduced points out to the specific area and not to the bucket, as you might risk deletion of all the upload areas otherwise. The upload area is highlighted in red in the example.
+
+**Prabhat: THIS IS VERY RISKY**
```

Comment:
This does sound like a good way of doing it, Once we have everything sorted I'll delete the other way and add this one as the way to do it

Reply:
Note to self: Delete `aws rm` command